samples/cellular/modem_shell/src/ppp/ppp_shell.c

```diff
@@ -151,41 +148,44 @@ static void ppp_shell_uart_conf_print(struct uart_config *ppp_uart_config)
 	mosh_print("  stop bits:    %s", tmp_str);
 }
 
-static int ppp_shell_cmd(const struct shell *shell, size_t argc, char **argv)
+static int cmd_ppp_up(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret = 0;
+
+	ret = ppp_ctrl_start();
+	if (ret >= 0) {
+		mosh_print("PPP up.");
+	} else {
+		mosh_error("PPP cannot be started");
+	}
+
+	return 0;
+}
+
+static int cmd_ppp_down(const struct shell *shell, size_t argc, char **argv)
+{
+	ppp_ctrl_stop();
+
+	return 0;
+}
+
+static int cmd_ppp_uartconf(const struct shell *shell, size_t argc, char **argv)
 {
 	int ret = 0;
 	bool uartconf_option_given = false;
+	struct uart_config ppp_uart_config;
 
 	ppp_shell_cmd_defaults_set(&ppp_cmd_args);
```

Comment:
I'm tempted to remove `struct ppp_shell_cmd_args_t` and put all variables into this function as local arguments.
Mainly because this structure is not passed to any other function so it's not really needed.
@jhirsi, what do you think?

Reply:
This is now done.